obj_gen.cpp

```diff
@@ -714,3 +714,84 @@ data_object* import_object_generator::get_object(int iter)
     return &m_object;
 }
 
+///////////////////////////////////////////////////////////////////////////
```

Comment:
is this line needed?

Reply:
I'm following the general style of this file, there's a separator line between all of the class implementations...